src/neo-vm/Types/Struct.cs

```diff
@@ -1,21 +1,26 @@
 using System.Collections.Generic;
 using System.Diagnostics;
-using System.Runtime.CompilerServices;
 
 namespace Neo.VM.Types
 {
     [DebuggerDisplay("Type={GetType().Name}, Count={Count}")]
     public class Struct : Array
     {
-        public Struct() : base() { }
+        public Struct(IEnumerable<StackItem> value = null)
```

Comment:
We don't need the `ReferenceCounter` ?

Reply:
There's another constructor.

Reply:
I need to review it more, but in which cases we need to create an array without the `ReferenceCounter`?

Reply:
If you need to create an array but not pushing it onto the stack, you can use this constructor to create it without reference counter. Maybe you need it for test purpose.